docs/conf.py

```diff
@@ -54,10 +54,11 @@
 # |version| and |release|, also used in various other places throughout the
 # built documents.
 #
-# The short X.Y version.
-version = '1.2.0'
-# The full version, including alpha/beta/rc tags.
-release = '1.2.0'
+with open('../version.txt', 'r') as reader:
```

Comment:
You may want to make that path relative to the `conf.py` file using os.path.relpath and __file__.

Reply:
Do you mean this?

```python
with open(os.path.relpath('../version.txt'), 'r') as reader:
```

I don't know why using `os.path.relpath` would be better, but I would appreciate it if you can enlighten me.

Reply:
Done in [83758d3](https://github.com/plone/ansible-playbook/pull/142/commits/83758d3f07925508d2c3b5510e92f25d220175d3).